coderd/notifications/manager_test.go

```diff
@@ -79,7 +81,7 @@ func TestBufferedUpdates(t *testing.T) {
 	// Wait for the expected number of buffered updates to be accumulated.
 	require.Eventually(t, func() bool {
 		success, failure := mgr.BufferedUpdatesCount()
-		return success == expectedSuccess && failure == expectedFailure
+		return success == expectedSuccess*2 && failure == expectedFailure*2 // Each message is enqueued twice.
```

Comment:
Your tests are going to become complicated once inbox can be disabled (by feature flag or otherwise).
I'd suggest creating a func which returns the configured targets and then you can `len` them instead of hardcoding magic numbers like this.

Reply:
I tried to clean it up a bit - still not perfect but as we do not have FF or experiment logic, please tell me if that's good enough.

Reply:
No I think we need to take this further; there are several other places relying on this magic number.